packages/toggle/src/Toggle.story.tsx

```diff
@@ -15,6 +15,7 @@ storiesOf('Toggle', module).add('Default', () => {
         padding: 20px;
       `}
     >
+      <div>Brooke</div>
```

Comment:
This probably wasn't an intentional addition.

Reply:
Yes already rm'd